src/app/containers/RadioSchedule/utilities/processRadioSchedule.js

```diff
@@ -20,6 +24,19 @@ export const getLink = (state, program, service) => {
   return state === 'live' ? `${url}/liveradio` : `${url}/${pid}`;
 };
 
+const logMissingFieldError = field => {
+  logger.error(
+    JSON.stringify(
+      {
+        event: RADIO_SCHEDULE_DATA_INCOMPLETE_ERROR,
+        message: `${field} field is missing in program`,
+      },
+      null,
+      2,
+    ),
+  );
+};
```

Comment:
This can be updated to have a simpler format:
```suggestion
  logger.error({
      event: RADIO_SCHEDULE_DATA_INCOMPLETE_ERROR,
      message: `${field} field is missing in program`,
    }),
  );
};
```

Reply:
I believe you were looking at our logs in the file `src/app/containers/RadioSchedule/Canonical/index.jsx`, but these are currently using an outdated format that needs updating.